bugsnag-android-unity/src/main/java/com/bugsnag/android/unity/UnityClient.java

```diff
@@ -19,6 +20,18 @@ public static void init(Context androidContext, String apiKey, boolean trackSess
         Configuration config = new Configuration(apiKey);
         config.setAutoCaptureSessions(trackSessions);
         client = new Client(androidContext, config);
```

Comment:
Possibly small issue: client initialization should be after setting all of the config properties in case of unexpected (background?) interactions between enabling session tracking and setting endpoints

Reply:
good point, changed